From d5f2c893f78acf44cf3f04d0606ca82a7fabbd45 Mon Sep 17 00:00:00 2001
From: Damian Hobson-Garcia <dhobsong@igel.co.jp>
Date: Thu, 13 Sep 2012 11:54:15 +0900
Subject: [PATCH 60/97] dfb-example/dfb-video-example: Move directfb.h include
 to after gst/gst.h

Due to a namespace conflict, including directfb.h before gst/gst.h causes
preprocessor expansion to tag some functions with incorrect attributes.
The attribute expands to __attribute__((__attribute__((no_instrument_funtion))))
, which is invalid and connot be compiled.
---
 ext/directfb/dfb-example.c       |    2 +-
 ext/directfb/dfb-video-example.c |    2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/ext/directfb/dfb-example.c b/ext/directfb/dfb-example.c
index 7fea97c..b361631 100644
--- a/ext/directfb/dfb-example.c
+++ b/ext/directfb/dfb-example.c
@@ -1,8 +1,8 @@
 
-#include <directfb.h>
 #include <stdio.h>
 #include <gst/gst.h>
 #include <libgen.h>
+#include <directfb.h>
 
 static IDirectFB *dfb = NULL;
 static IDirectFBSurface *primary = NULL;
diff --git a/ext/directfb/dfb-video-example.c b/ext/directfb/dfb-video-example.c
index 44d4050..fa79ee1 100644
--- a/ext/directfb/dfb-video-example.c
+++ b/ext/directfb/dfb-video-example.c
@@ -1,7 +1,7 @@
 
-#include <directfb.h>
 #include <gst/gst.h>
 #include <libgen.h>
+#include <directfb.h>
 
 static IDirectFB *dfb = NULL;
 static IDirectFBSurface *primary = NULL;
-- 
1.7.10.4

